### ISSUE TITLE
Interference between toctree.numbered and :tocdepth: in HTML

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1251]
I attached a full example of this buggy behaviour.
What must be noticed is that:

in index.rst, I use the :numbered: attribute in the toctree :

```
#!rst

  .. toctree::
    :numbered:

    foo
    bar
```

and in bar.rst, I use the :tocdepth: directive :

```
#!rst


  :tocdepth: 2

  ===
  Bar
  ===

  ...
```

and all sections in bar.rst _below_ level 2 are wrongly numbered "2" instead of "2.x.y" etc...

Note that limiting the depth of the numbering (by providing a numeric argument to :numbered:) does not solve the problem... Those buggy sections are still numbered, and still incorrectly.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1251
- Originally reported by: Pierre-Antoine Champin
- Originally created at: 2013-08-30T10:59:55.736

**Comment by shimizukawa** [comment]
_From Andrea Cassioli on 2014-07-31 09:44:35+00:00_

Hi same behavior with Sphinx 1.2.2. Any news on that?

Hi use :tocdepth: to limit the toc tree in the side bar.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-08-02 09:06:04+00:00_

Confirmed. I'll fix it.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-08-02 09:11:28+00:00_

When specifying toctree :numbered: option and :tocdepth: metadata, sub section number that is larger depth than `:tocdepth:` is shrinked. Closes #1251.

→ <<cset da4c67e2a35ef84395a0d6cee84a13b661c458c0>>

**Comment by shimizukawa** [comment]
_From [JasonGao180](https://bitbucket.org/JasonGao180) on 2014-10-18 08:12:00+00:00_

Will this issue been fixed in 1.2.3?

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-10-18 11:05:30+00:00_

[JasonGao180](https://bitbucket.org/JasonGao180) No. This change has been shipped with Sphinx-1.3b1.

**Comment by shimizukawa** [comment]
_From [JasonGao180](https://bitbucket.org/JasonGao180) on 2014-10-21 04:22:23+00:00_

[Takayuki Shimizukawa](https://bitbucket.org/shimizukawa) I installed Sphinx-1.3b1, and build the html from source in secnum.tar.gz. It seems handle the tocdepth incorrectly.

HTML
![QQ20141021-1@2x.png](https://bitbucket.org/repo/7zgG/images/1897682775-QQ20141021-1@2x.png)

Source
![QQ20141021-2@2x.png](https://bitbucket.org/repo/7zgG/images/2969499815-QQ20141021-2@2x.png)

**Comment by shimizukawa** [comment]
_From [JasonGao180](https://bitbucket.org/JasonGao180) on 2014-10-21 08:14:01+00:00_

It seems that the Bar is convert to a compact_paragraph. And the _toctree_prune will ignore it.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-12-04 08:40:22+00:00_

[JasonGao180](https://bitbucket.org/JasonGao180) confirmed. I'll fix it. Thanks for reporting!

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-12-05 10:06:54+00:00_

Fix again. Sections which depth are lower than :tocdepth: should not be shown on localtoc sidebar. Closes #1251

→ <<cset edf63e1406a79be78c1285648621ecf77b4f6fc3>>
